### ISSUE TITLE
Join room not work

### ISSUE THREAD

**Issue by nvcken** [wamoyo/webrtc-cafe#2]
tested on Chrome

**Comment by wamoyo** [comment]
Yeah, I recall there being some kind of Chrome vs Firefox discrepancy in WebRTC implementation. These guys have an excellent library that helps with that: https://about.talky.io/core/
